content/docs/04.5-devx/03-cluster-groups.md

```diff
@@ -101,6 +101,61 @@ To delete a cluster group, select `Delete Cluster form the **Settings** and ente
 * Azure
 * AKS
 * VMWare
+* OpenShift
+
+<Tabs>
```

Comment:
Is the idea to add other platforms in the future?  I guess the first question is, how much different are the other platforms? It just feels a bit awkward with a single tab.

Reply:
@karl-cardenas-coding As we discussed, I removed single tab and made it a subsection.